Makefile

```diff
@@ -102,15 +103,15 @@ run/pipecd:
 	yarn --cwd web build
 
 	@echo "Building docker image and pushing it to local registry..."
-	docker build -f cmd/pipecd/Dockerfile -t localhost:5001/pipecd:$(BUILD_VERSION) .
-	docker push localhost:5001/pipecd:$(BUILD_VERSION)
+	docker build -f cmd/pipecd/Dockerfile -t $(DOCKER_REGISTRY)/pipecd:$(BUILD_VERSION) .
```

Comment:
I don't think this fix the requirement since run/pipecd always build a new image container  on run, so I guess we still can't use previous image for debugging 🤔

Reply:
I see...
Thank you.